src/components/Stakeholders/Add.js

```diff
@@ -0,0 +1,357 @@
+                helperText={errors.email}
+                className={classes.textField}
+                label="Email"
+                variant="outlined"
+                name="email"
+                type="email"
+                onChange={handleChange}
+                value={data.email}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                error={!!errors?.phone}
+                helperText={errors.phone}
+                className={classes.textField}
+                label="Phone"
+                variant="outlined"
+                name="phone"
+                type="tel"
+                onChange={handleChange}
+                value={data.phone}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Website"
+                variant="outlined"
+                name="website"
+                type="url"
+                onChange={handleChange}
+                value={data.website}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Logo URL"
+                variant="outlined"
+                name="logo_url"
+                type="url"
+                onChange={handleChange}
+                value={data.logo_url}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Map"
+                variant="outlined"
+                name="map"
+                placeholder="map_address"
+                onChange={handleChange}
+                value={data.map}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+          </Grid>
+          <Grid container justify-content="space-between">
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <FormLabel id="relation">Relationship</FormLabel>
+              <RadioGroup
+                aria-labelledby="radio-buttons-group-label"
+                name="relation"
+                value={data.relation}
+                className={classes.radioGroup}
+                onKeyDown={handleEnterPress}
+              >
+                <FormControlLabel
+                  control={
+                    <Radio
+                      label="Parent"
+                      value="parents"
+                      onChange={handleChange}
+                    />
+                  }
+                  label="Parent"
```

Comment:
It's not clear whether _Parent_ means that this stakeholder is the parent or the ID entered here is the parent.
It'd be clearer to replace this with an option to _Link to parent_ that presents a list of stakeholders to link this one to.
Realistically, I only see us linking new stakeholders to their parents at the point of creation.